src/compiler/tsc.ts

```diff
@@ -120,7 +120,7 @@ namespace ts {
                 createWatchOfConfigFile(configParseResult, commandLineOptions);

```

Comment:
I am assuming you arent adding this portion in this PR. Can you add a TODO and create a issue tracking that. Thanks